README.md

```diff
@@ -48,6 +52,34 @@ Crashes will be saved to disk when they occur, then reported to Rollbar the next
 
 Rollbar uses [PLCrashReporter](https://www.plcrashreporter.org/) to capture uncaught exceptions and fatal signals. Note that only one crash reporter can be active per app. If you initialize multiple crash reporters (i.e. Rollbar alongside other services), only the last one initialized will be active.
 
+### Swift
+
+Importing with Swift requires the additional step of adding the following lines to your Bridging-Header file:
+
+```c
+#import <SystemConfiguration/SystemConfiguration.h>
+#import <Rollbar/Rollbar.h>
+```
+
+If you have no Bridging Header file, the easiest way to correctly configure it is to add an objective-c (`.m`) file. When you do so, XCode will prompt you to create a briding header file, and will configure your build environment to automatically include those headers in all your Swift files. After creating the Briging-Header file, you can delete the objective-c file.
```

Comment:
Is it enough to add an empty `.m` file? Maybe we should have the instructions say something like "add an empty file called `dummy.m`"